packages/block-library/src/column/edit.js

```diff
@@ -97,7 +111,7 @@ function ColumnEdit( {
 					/>
 				</PanelBody>
 			</InspectorControls>
-			<div { ...innerBlocksProps } />
+			<div { ...innerBlocksProps } aria-label={ label } />
```

Comment:
It's better to add the `aria-label` property in `useInnerBlocksProps` to use the existing mechanism for props and options.

```
const innerBlocksProps = useInnerBlocksProps(
	{ ...blockProps, 'aria-label': label },
	{
		templateLock,
		renderAppender: hasChildBlocks
			? undefined
			: InnerBlocks.ButtonBlockAppender,
	}
);
```

Reply:
@ntsekouras Not sure if I can do that in this case. I'm reusing the `aria-label` prop from `innerBlocksProps` to create the new label:
```js
	const label = sprintf(
		/* translators: 1: Block label (i.e. "Block: Column"), 2: Position of the selected block, 3: Total number of sibling blocks of the same type */
		__( '%1$s (%2$d of %3$d)' ),
		innerBlocksProps[ 'aria-label' ],
		currentColumnPosition,
		columnsCount
	);
```

Reply:
The above snippet will just change to use `blockProps[ 'aria-label' ]` instead of `innerBlocksProps[ 'aria-label' ]`. Your code will just need some reordering then.

Reply:
Ah yeah! I thought the original label was coming from the `useInnerBlockProps` hook. Made the change.